infrastructure/infrastructure.py

```
@@ -0,0 +1,476 @@
+def create_swarm(instance: Instance) -> tuple[str, str]:
+    with Connection(
+        host=instance.public_ip,
+        user="ubuntu",
+        connect_kwargs={"key_filename": str(instance.key_path), "banner_timeout": 30, "auth_timeout": 30},
+        connect_timeout=10,
+    ) as ssh:
+        try:
+            result = ssh.run(f"docker swarm init --advertise-addr {instance.tailscale_ip}", warn=True)
+            if result.failed:
+                print("Node already part of a swarm, continuing...")
+        except Exception as e:
```

Ruff (0.13.0)

Do not catch blind exception: `Exception`

(BLE001)